capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.capitalize
+    puts @nameable.correct_name.capitalize
```

Comment:
- [ ] please remove the puts statement you included here for testing purposes.

Reply:
Remove puts statement